node/network/dispute-distribution/src/sender/mod.rs

```diff
@@ -366,6 +366,13 @@ async fn get_active_session_indices<Context>(
 	// Iterate all heads we track as active and fetch the child' session indices.
 	for head in active_heads {
 		let session_index = runtime.get_session_index_for_child(ctx.sender(), *head).await?;
+		// Cache session info
+		match runtime.get_session_info_by_index(ctx.sender(), *head, session_index).await {
+			Ok(_) => { /* do nothing */ },
+			Err(err) => {
+				gum::warn!(target: LOG_TARGET, ?err, ?session_index, "Can't cache SessionInfo");
```

Comment:
Is this really a warning for the user? Wouldn't be a debug message be better?

Reply:
My reasoning is `SessionInfo` should be available at this point so if we can't fetch it it's worth spamming the log.

I agree that from user perspective it's not that important though.